### PR TITLE
Added recursive option

### PR DESCRIPTION
**Pull request by Icare67**
I needed to recursively convert a complete hierarchy of directories. I added the recursive option for my needs. Take it if you are interested.

### CODE REVIEW

File: RptToXml/Program.cs [hunk 1]
```diff
@@ -12,44 +12,48 @@ static void Main(string[] args)
 		{
 			if (args.Length < 1)
 			{
-				Console.WriteLine("Usage: RptToXml.exe <RPT filename | wildcard> [outputfilename]");
+				Console.WriteLine("Usage: RptToXml.exe < -r | RPT filename | wildcard> [outputfilename]");
```
Comment (ajryan) [inline RptToXml/Program.cs:15]: Can we add a line to the usage output that describes how `-r` works?

File: RptToXml/Program.cs [hunk 2]
```diff
@@ -12,44 +12,48 @@ static void Main(string[] args)
...
+				var directory = Path.GetDirectoryName(rptPathArg);
+				if (String.IsNullOrEmpty(directory))
+				{
+					directory = ".";
+				}
+				var matchingFiles = Directory.GetFiles(directory, searchPattern: Path.GetFileName(rptPathArg));
+				rptPaths.AddRange(matchingFiles.Where(ReportFilenameValid));
+				if (rptPaths.Count == 0)
+				{
+					Trace.WriteLine("No reports matched the wildcard.");
```
Comment (ajryan) [inline RptToXml/Program.cs:48]: This should be console.writeline, not trace - we ahven't hooked up the trace listener yet.

Comment (ajryan) [reply RptToXml/Program.cs:48]: Or move the tracelistener init aabove rptPAths init as it was before

**Comment by ajryan** [comment]
Thanks for this! Just a few comments.

**Review by ajryan — CHANGES_REQUESTED** [review]

**Comment by Icare67** [comment]
Here are the requested changes

**Review by ajryan — APPROVED** [review]
Thanks for this!

**Pull request merged**